R/wrap_gradient8.R

```diff
@@ -0,0 +1,21 @@
+#' wrap_gradient8
+#'
+#' This will make the Gradient8 funktion available to R from the libtotopotoolbox subdirectory
+#' 
+#' @param dem Input of type Spatrast from Terra
+#' @param use_mp Future feature that will allow to parallelize the code
+#'
+#' @return A Spatrast
+#' @export
+
+wrap_gradient8 <- function(dem,use_mp=0) {
```

Comment:
Let's change the name of the R function to `gradient8`. This is a better name for the user-facing function and it matches with our MATLAB and Python interface. 

I think usually when we wrap a libtopotoolbox function called `foo`, our C wrapper in this repository should be called `wrap_foo` to avoid name collisions with the libtopotoolbox function and the corresponding R function should be called `foo`. There are a few exceptions to this rule, and I'll let you know when we need to use a different naming convention.

Reply:
done